assets/airports/ltba.json

```diff
@@ -1,254 +1,6868 @@
+            ]
+        },
+        "EDASA1Z": {
+            "icao": "EDASA1Z",
+            "name": "Edasa One Zulu",
+            "rwy": {
+                "LTBA35R": []
+            },
+            "body": [["BA010", "A16+|S240-"], "BKZ", ["BA044", "A120-"], "BAVUL", "ULPER", "EKDAM", "BRY", "EDASA"],
+            "exitPoints": {
+                "EDASA": []
+            },
+            "draw": [
+                ["_RWY17L", "BA010", "BKZ", "BA044", "BAVUL", "ULPER", "EKDAM", "BRY", "EDASA"]
+            ]
+        },
+        "EKI1D": {
+            "icao": "EKI1D",
+            "name": "Tekirdag One Delta",
+            "rwy": {
+                "LTBA35L": []
+            },
+            "body": [["^_IST356007", "A16+"], "_EKI080055.5", "EKI"],
+            "exitPoints": {
+                "EKI": []
+            },
+            "draw": [
+                ["_RWY17R", "_IST356007", "_EKI080055.5", "EKI"]
+            ]
+        },
+        "EKI1K": {
+            "icao": "EKI1K",
+            "name": "Tekirdag One Kilo",
+            "rwy": {
+                "LTBA35R": []
+            },
+            "body": [["^_IST356007", "A16+"], "_EKI080055.5", "EKI"],
+            "exitPoints": {
+                "EKI": []
+            },
+            "draw": [
+                ["_RWY17L", "_IST356007", "_EKI080055.5", "EKI"]
+            ]
+        },
+        "EKI1N": {
+            "icao": "EKI1N",
+            "name": "Tekirdag One November",
+            "rwy": {
+                "LTBA05": []
+            },
+            "body": [["^_IST051004.5", "A10+"], "CEK", "EKI"],
+            "exitPoints": {
+                "EKI": []
+            },
+            "draw": [
+                ["_RWY23", "_IST051004.5", "_CEK083016", "CEK", "EKI"]
+            ]
+        },
+        "EKI2G": {
+            "icao": "EKI2G",
+            "name": "Tekirdag Two Golf",
+            "rwy": {
+                "LTBA23": []
+            },
+            "body": [["^_IST245005", "A14+"], "_IST267009", "EKI"],
+            "exitPoints": {
+                "EKI": []
+            },
+            "draw": [
+                ["_RWY05", "_IST245005", "_IST267009", "EKI"]
+            ]
+        },
+        "EKI2S": {
+            "icao": "EKI2S",
+            "name": "Tekirdag Two Sierra",
+            "rwy": {
+                "LTBA17L": [],
+                "LTBA17R": []
+            },
+            "body": [["^_IST177005", "A14+"], "_IST267009", "EKI"],
+            "exitPoints": {
+                "EKI": []
+            },
+            "draw": [
+                ["_RWY35R", "_IST267009", "EKI"],
+                ["_RWY35L", "_IST267009", "EKI"]
+            ]
+        },
+        "ERSEN1P": {
+            "icao": "ERSEN1P",
+            "name": "Ersen One Papa",
+            "rwy": {
+                "LTBA05": []
+            },
+            "body": [["BA035", "A16+|S220"], "BKZ", "TEMGO", "YASEN", "BA046", "KUBAP", "BEBEG", "ERSEN"],
```

Comment:
Remove TEMGO. While the SID passes TEMGO, the textual description does not include TEMGO as part of the procedure.